webpack.config.js

```diff
@@ -81,12 +81,13 @@ const webpackConfig = {
                     '/dhis-web-mapping/**',
                     '/dhis-web-event-reports/**',
                     '/dhis-web-event-visualizer/**',
+                    '/dhis-web-core-resource/**',
+                    '/dhis-web-maps/**'
```

Comment:
Does this app use the maps plugin?

Reply:
@varl At least its included in index.html as ../dhis-web-maps/map.js, but I have not checked why every script is included except fixing so there's no 404's when running locally.